### DISCUSSION TITLE
Is it possible to only get user account address without a websocket connection?

### DISCUSSION THREAD

**Discussion by waghmare-omkar** [WalletConnect/walletconnect-monorepo#1278] [Developer Support]
I just want a user to share their public account address to my website. I do not want all other jazz and create and maintain a websocket connection. Is there an easy way to just get account number by making a user scan a QR code and then sign a transaction to share their public address and balance with me one-time?

If not wallet connect, is there anything else that i could use?

**Comment by JeneaVranceanu** [comment]
The mobile app or some other kind of application must know how to communicate with your website. 
Typically it's resolved by using a web socket connection. There's no need to use wallet connect for that. 

Just off the top of my head - you could use Firebase Realtime Database but I wouldn't say it's significantly easier/difficult. And most likely it's using web sockets inside too.

**Reply by chadyj** [reply]
WalletConnect is working on a new SDK exactly for this use case.  It is called WalletConnect Auth.

You can find more about it here https://docs.walletconnect.com/2.0/specs/auth/

> Auth API provides a simple and lean interface to enable applications to authenticate wallet users to login/signin into applications with their wallets by automatically an authentication message.

There are a few caveats:

- Not released yet. But look out for an alpha in a few days
- Is v2 only, and Wallet support is still rolling out
